### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Urchmaney**
Requesting pull request

### CODE REVIEW

**Comment by Torres-ssf** [comment]
- [ ]  There are some errors in your CSS code. Check them using this CSS [validator](https://jigsaw.w3.org/css-validator/) . Ignore the warnings.

- [ ] Try to make your page fit the full width of the page(like the original one):
image


#### Navbar

- [ ] It's best practice to use the `nav` tag for the navbar.

#### Main content links(The 3 main sentences)

- [ ] There should be three links
- [ ] Each link is an `a` HTML tag, with a `div` tag inside it.
- [ ] The `div` tag inside the `a` tag should use an image as background. The text on the original page is actually an image
- [ ] It's best practice to the `main` tag for the main content of your page.

#### Footer

- [ ] The footer must start at the 4 iphone promo images(top part of the footer)
- [ ] The top part is made of four `a` tags with `div` tags inside it
- [ ] The `div`s inside the `a` tags should use an image as a background

File: index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    </title>
+</head>
+
+<body>
+    <div id="container">
+
+        <div id="header">
+            <div id="header-container">
+                <nav>
+                    <ul id="nav-bar">
```
Comment (Ispirett) [inline index.html:19]: You should use floats or display: flex for this section instead of inline-block

File: index.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                    </a>
+
+                </main>
+
+
+            </div>
+        </div>
+
+
+        <div id="footer">
```
Comment (Ispirett) [inline index.html]: Use a footer tag instead of a div here

File: index.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css" />
+    <title>
+        Background and Gradients
+    </title>
+</head>
+
+<body>
+    <div id="container">
```
Comment (Ispirett) [inline index.html]: Use the main tag instead of div here

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey @Urchmaney  great work fixing the issues form you last **PR**  your site really looks gorgeous :100: 
However there are  still some issues that need addressing.
 - [ ] The search field should be an <input> HTML tag with the focus effect enabled

**Comment by Torres-ssf** [comment]
### Great work 👏
You can merge to master now. Don't forget to submit the form. Good luck with the next project

**Pull request merged**